shihabrafio/Movie-Monday
# Home page and Popup comments window for Movie-Monday using TV-Maze & Involvement APIs

Added features:

-  Added Header
-  Added Footer
-  Home Page
    -  GET elements from TV-Maze API and list them.
    -  POST/GET likes into/from Involvement API
    -  Implement the addLike button.
    -  Count items
    -  Test items counter

-  Popup comments window
     -  Add more details from TV-Maze API
     -  POST/GET comments into/from Involvement API
     -  Add Comment
     -  Count Comments
     -  Test comments counter

-  Added README file

  See the [Presentation Video](https://drive.google.com/file/d/1iPTBRkNIa2iV9eNqqHPA050MIDzOtzcs/view?usp=sharing)
  
Follow the given design

Try the [Live Demo](https://shihabrafio.github.io/Movie-Monday/)
